frontend/src/app/patients/ManagePatients.test.tsx

```diff
@@ -85,38 +92,248 @@ describe("ManagePatients", () => {
 
 const patients = [
```

Comment:
(no-op) Holy cow, thanks for adding all these patients. Makes me want to up the priority of this ticket: #2929